nix/deps/gradle/generate.sh

```diff
@@ -83,6 +83,10 @@ fi
 # Run each stage in order
 gen_proj_list
 gen_deps_list
+# FIXME: Temporary fix for missing package.
+echo 'org.jetbrains.kotlin.jvm:org.jetbrains.kotlin.jvm.gradle.plugin:1.6.10' >> "${DEPS_LIST}"
+echo 'com.android.tools.lint:lint-gradle:30.3.0' >> "${DEPS_LIST}"
+echo 'com.android.tools.build:gradle:7.0.4' >> "${DEPS_LIST}"
```

Comment:
Yes, we will definitely need a proper fix for this, and theoretically that will be also a solution to this:

* https://github.com/status-im/status-mobile/issues/15447